feature-wallet-impl/src/main/java/jp/co/soramitsu/feature_wallet_impl/data/repository/WalletRepositoryImpl.kt

```diff
@@ -190,33 +204,29 @@ class WalletRepositoryImpl(
         return accountsByAddress[accountAddress]?.name
     }
 
-    private fun createTransaction(hash: String, transfer: Transfer, senderAddress: String, fee: BigDecimal) =
-        Transaction(
+    private fun defineAccountNameForTransaction(
+        accountsByAddress: Map<String, WalletAccount>,
+        displayAddress: String?
+    ): String? {
+        if (displayAddress == null) return null
+        return accountsByAddress[displayAddress]?.name
+    }
+
+    private fun createOperation(hash: String, transfer: Transfer, senderAddress: String, fee: BigDecimal, source: OperationLocal.Source) =
```

Comment:
probably it should be named createTransfer, since we're creating Transfer here

Reply:
But I am returning OperationLocal from this method. I now think maybe it should be called createOperationLocal

Reply:
Yes, but logically this method is used to create a Transfer object to save into db. I mean, it cannot be used to create another type of Operation and thus name is confusing

Reply:
Done